wisdem/rotorse/rotor_structure.py

```diff
@@ -1893,26 +1893,33 @@ def setup(self):
             "yu_te",
             "yl_te",
         ]
+
         self.add_subsystem("strains", ComputeStrains(modeling_options=modeling_options), promotes=promoteListStrains)
         self.add_subsystem("tip_pos", TipDeflection(), promotes=["tilt", "pitch_load"])
         self.add_subsystem(
             "aero_hub_loads",
             CCBladeEvaluate(modeling_options=modeling_options),
             promotes=promoteListAeroLoads + ["presweep", "presweepTip"],
         )
-        self.add_subsystem(
-            "constr", DesignConstraints(modeling_options=modeling_options, opt_options=opt_options), promotes=["s"]
-        )
-        self.add_subsystem("brs", BladeRootSizing(rotorse_options=modeling_options["WISDEM"]["RotorSE"]))
-        if modeling_options["WISDEM"]["RotorSE"]["bjs"]:
+        if not modeling_options["WISDEM"]["RotorSE"]["user_elastic"]:
+            # missing spars_tereinf for using this ComputeStrains for blade with user defined elastic properties
+
+            # TODO YL: can we find ways to set default or still compute constraints for user defined blade? 
```

Comment:
I think this comment is outdated. I don't think we can compute strains, nor should we, if the user is passing elastic properties for the blade